tracer/src/Datadog.Trace/Configuration/ImmutableTracerSettings.cs

```diff
@@ -247,6 +263,22 @@ public ImmutableTracerSettings(TracerSettings settings)
         /// <seealso cref="ConfigurationKeys.ExpandRouteTemplatesEnabled"/>
         internal bool ExpandRouteTemplatesEnabled { get; }
 
+        /// <summary>
+        /// Gets a value indicating the regex to apply to obfuscate http query strings.
+        /// </summary>
+        internal string ObfuscationQueryStringRegex { get; }
+
+        /// <summary>
+        /// Gets a value indicating whether or not http.url should contain the query string, enabled by default with DD_HTTP_SERVER_TAG_QUERY_STRING
+        /// </summary>
+        internal bool EnableQueryStringReporting { get; }
```

Comment:
If you accept [my suggestion](https://github.com/DataDog/dd-trace-dotnet/pull/2915/files#r919466741), you should update this as well